pyhf/tensor/jax_backend.py

```diff
@@ -13,7 +14,7 @@ class jax_backend(object):
     def __init__(self, **kwargs):
         self.name = 'jax'
 
-    def clip(self, tensor_in, min, max):
+    def clip(self, tensor_in, min_value, max_value):
```

Comment:
@kratsg This isn't strictly necessary, as I have control over the codefactor checker so I just turned them off for this PR. So if you don't want to have this backen's functions be slightly different then you can revert/drop this.

Reply:
I changed all backends to match the signature. I think it's probably cleaner if we do it this way...

Reply:
Okay. I'm fine with it.